src/Components/Search/NewSearch/Mobile/SearchResultsList.tsx

```diff
@@ -0,0 +1,247 @@
+import { FC, useState } from "react"
+import {
+  RelayPaginationProp,
+  createPaginationContainer,
+  graphql,
+} from "react-relay"
+import { SearchResultsList_viewer$data } from "__generated__/SearchResultsList_viewer.graphql"
+import { extractNodes } from "Utils/extractNodes"
+import {
+  NewSuggestionItem,
+  SuggionItemOptionProps,
+} from "Components/Search/NewSearch/SuggestionItem/NewSuggestionItem"
+import { Flex, Spinner } from "@artsy/palette"
+import { SystemQueryRenderer } from "System/Relay/SystemQueryRenderer"
+import { SearchResultsListQuery } from "__generated__/SearchResultsListQuery.graphql"
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import {
+  SearchNodeOption,
+  formatOptions,
+} from "Components/Search/NewSearch/utils/formatOptions"
+import { SeeFullResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/SeeFullResults"
+import { InfiniteScrollSentinel } from "Components/Search/NewSearch/Mobile/SearchResultsList/InfiniteScrollSentinel"
+import { ContentPlaceholder } from "Components/Search/NewSearch/Mobile/SearchResultsList/ContentPlaceholder"
+import { NoResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/NoResults"
+
+interface SearchResultsListProps {
+  relay: RelayPaginationProp
+  viewer: SearchResultsList_viewer$data
+  query: string
+  onClose: () => void
+}
+
+const ENTITIES_PER_SCROLL = 10
+
+const SearchResultsList: FC<SearchResultsListProps> = ({
+  relay,
+  viewer,
+  query,
+  onClose,
+}) => {
+  const tracking = useTracking()
+  const [isLoading, setIsLoading] = useState(false)
+
+  const options = extractNodes(viewer.searchConnection)
+
+  if (!relay.isLoading() && options.length === 0) {
+    return <NoResults query={query} mt={50} mx={20} />
```

Comment:
I think so, at least this way it is close to design specs

![Screenshot 2023-05-17 at 10 44 30](https://github.com/artsy/force/assets/3934579/fbe74b19-b01f-4206-9d1c-74683f00aca4)

Reply:
Got it. In that case I think it would be better to use one of the spacing values from [here](https://palette.artsy.net/atoms/spacing/#all-spacing-values). E.g. instead of `mt={50}` use `mt={4}` or `mt={6`}, and `mx={2}` instead of `mx={20}`